package.json

```diff
@@ -53,12 +53,15 @@
     "express": "^4.16.4",
     "express-ipfilter": "^1.0.1",
     "http-auth": "^3.2.3",
+    "install": "^0.13.0",
```

Comment:
What's this?

Reply:
This was attached when I installed `parse-domain`

Reply:
I deleted it, and parse domain still works :)